examples/tree_1d_dgsem/elixir_maxwell_E_excitation.jl

```diff
@@ -0,0 +1,57 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection equation
+
+equations = MaxwellEquations1D()
+
+solver = DGSEM(polydeg = 3, surface_flux = flux_lax_friedrichs)
+
+coordinates_min = 0.0
+coordinates_max = 1.0
+
+# Create a uniformly refined mesh with periodic boundaries
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 4,
+                n_cells_max = 30_000) # set maximum capacity of tree data structure
+
+# Excite the electric field which causes a standing wave 
+# The solution is an undamped exchange between electric and magnetic energy            
+function initial_condition_E_excitation(x, t, equations::MaxwellEquations1D)
+    c = equations.speed_of_light[1]
+    E = -c * sin(2 * pi * x[1])
+    B = 0.0
+
+    return SVector(E, B)
+end
+
+initial_condition = initial_condition_E_excitation
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+ode = semidiscretize(semi, (0.0, 1e-7));
+
```

Comment:
This is a very short run time. Is this due to the fast propagation? Would it then make sense to change `dt` in the solver to $10^{-7}$?

Reply:
Could you please add such a comment to the elixir?